spec/features/user_invites_friend_spec.rb

```diff
@@ -27,18 +27,24 @@ def cas_accepts_invitation
     
     fill_in "Password", with: "password"
     fill_in "Full Name", with: "Castiel"
-    click_button "Submit"
+    fill_in "Credit Card Number", with: "4242424242424242"
+    fill_in "Security Code", with: "123"
+    select "11 - November", from: "date_month"
+    select "2019", from: "date_year"
+    click_button "Sign Up"
+    expect(page).to have_content("Profile was successfully created. Please sign in.")
   end
   
   def cas_signs_in
     fill_in "Email Address", with: "cas@example.com"
     fill_in "Password", with: "password"
     click_button "Submit"
+    expect(page).to have_content("You're signed in.")
   end
   
   def cas_should_follow(dean)
```

Comment:
This method doesn't need the argument. Since you do have the user though, you could also use `dean.name` instead of hard coding a name. I would recommend using less specific variable names though. `user` would be better.